R/TealAppDriver.R

```diff
@@ -257,6 +250,35 @@ TealAppDriver <- R6::R6Class( # nolint: object_name.
       available_datasets[displayed_datasets_index]
     },
     #' @description
+    #' Test if CSS elements are visible on the page via Javascript call.
+    #' @param selector (`character`) CSS selector to test visibility.
+    #' A `CSS` id will return only one element if the UI is well formed.
+    #' @param content_visibility_auto,opacity_property,visibility_property (`logical`) See more information
```

Comment:
```suggestion
    #' @param content_visibility_auto,opacity_property,visibility_property (`logical(1)`) See more information
```

Reply:
Where is this `more` information : P ?

Reply:
The more information is on the line below

`#' on <https://developer.mozilla.org/en-US/docs/Web/API/Element/checkVisibility>.`